### DISCUSSION TITLE
Welcome to PeerTuber Discussions!

### DISCUSSION THREAD

**Discussion by MikeHuntington** [PeerTuber/PeerTuber#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhk2** [comment]
Hola,

Previously I did some work with the Thorium android client and a PeerTube client for Android TVs. Recently I've been concentrating on Plug-in development, primarily with Lighting payments and Podcast compatibility.
